README.md

```diff
@@ -7,9 +7,7 @@ White-listing of URLs is possible for both.
 
 ## Note
 
-This library is currently intended for use by the Agent Services team.
-It has a direct dependency on http-verbs which makes the upgrade path to Play 2.6 or later more difficult.
-Until this is resolved we and PlatOps recommend that other teams avoid adding new dependencies on agent-kenshoo-monitoring.
+This library is currently intended for use by the Agent Services team. Since v4.3.0, to support Play 2.6 migrations the library no longer has a dependency on HttpVerbs. Now, you'll need an instance of HttpClient defined in scope.
```

Comment:
and mention about implicit ExecutionContext too...
Also, no one cares about the README text i guess  :)

Reply:
> and mention about implicit ExecutionContext too...
> Also, no one cares about the README text i guess :)

I care about the README - please keep it up to date :)